api/v01/vrp.rb

```diff
@@ -169,7 +169,7 @@ class Vrp < APIBase
               present(OptimizerWrapper.build_csv(solution[:result]), type: CSV)
             else
               present({
-                solutions: [solution[:result]].flatten(1),
+                solutions: solution[:result].is_a?(Array) ? solution[:result] : [solution[:result]],
```

Comment:
I am glad this improved the performance as expected. @fonsecadeline if we don't have a ticket can you create one about this issue please, solution[:result] should be always an array whether coming from Redis.get, wrapper_vrp or else.

Reply:
https://gitlab.com/mapotempo/optimizer-api/-/issues/782